inc/classes/preload/class-full-process.php

```diff
@@ -130,7 +130,7 @@ public function complete() {
 	 * @see WP_Background_Process::is_process_running()
 	 * @return boolean
 	 */
-	public function is_process_running() {
+	public function is_process_running() { // phpcs:ignore Generic.CodeAnalysis.UselessOverridingMethod.Found
 		return parent::is_process_running();
 	}
 }
```

Comment:
We probably can remove this method instead of using an ignore, it's not doing anything special